### ISSUE TITLE
Add Checkpoint CP 1450

### ISSUE THREAD

**Issue by Geo-Ron** [fusioninventory/sysobject.ids#32]
The device is not correctly identified.

**Output of snmpget:**
5-12-2018 13:35:38 (1 ms) : Device: <IP>
5-12-2018 13:35:38 (1 ms) : SNMP V2c
5-12-2018 13:35:38 (2 ms) : Custom OID 1.3.6.1.2.1.1.2.0
5-12-2018 13:35:39 (216 ms) : SNMP Datatype: ASN_OBJECT_ID
5-12-2018 13:35:39 (217 ms) : -------
5-12-2018 13:35:39 (218 ms) : Value: 1.3.6.1.4.1.8072.3.2.10
5-12-2018 13:35:39 (219 ms) : Done

**Output of netinventory:**
fusioninventory-netinventory.bat --credentials version:2c,community:OURCOMMUNITY --debug --host XXX.XXX.XXX.XXX
[debug] creating 1 worker threads
[debug] [thread 1] creation
[debug] [thread 1] scanning 0: XXX.XXX.XXX.XXX
[debug] full match for sysobjectID .1.3.6.1.4.1.8072.3.2.10 in database
[debug] [thread 1] termination
```
<?xml version="1.0" encoding="UTF-8" ?>
<REQUEST>
  <CONTENT>
    <DEVICE>
      <FIRMWARES>
        <DESCRIPTION>CheckPoint SVN version</DESCRIPTION>
        <MANUFACTURER>CheckPoint</MANUFACTURER>
        <NAME>SVN Foundation</NAME>
        <TYPE>system</TYPE>
        <VERSION>R77.20</VERSION>
      </FIRMWARES>
      <INFO>
        <COMMENTS>Linux %HOSTNAME% 3.10.20-al-5.0-pr2 #1 SMP Wed May 9 14:34:10 ID
T 2018 armv7l</COMMENTS>
        <CONTACT>Personen Name</CONTACT>
        <ID>0</ID>
        <IPS>
          <IP>XXX.XXX.XXX.XXX</IP>
          <IP>XXX.XXX.XXX.XXX</IP>
          <IP>127.0.0.1</IP>
        </IPS>
        <LOCATION>Our Office</LOCATION>
        <MAC>xx:xx:xx:xx:xx:xx</MAC>
        <MANUFACTURER>CheckPoint</MANUFACTURER>
        <MEMORY>0</MEMORY>
        <MODEL>CP 1450</MODEL>
        <NAME>%HOSTNAME%</NAME>
        <RAM>0</RAM>
        <TYPE>NETWORKING</TYPE>
        <UPTIME>21 days, 20:55:11.49</UPTIME>
      </INFO>
      <PORTS>
        <PORT>
          <IFDESCR>lo</IFDESCR>
          <IFINERRORS>0</IFINERRORS>
          <IFINOCTETS>520452463</IFINOCTETS>
          <IFINTERNALSTATUS>1</IFINTERNALSTATUS>
          <IFLASTCHANGE>0.00 seconds</IFLASTCHANGE>
          <IFMTU>65536</IFMTU>
          <IFNAME>lo</IFNAME>
          <IFNUMBER>1</IFNUMBER>
          <IFOUTERRORS>0</IFOUTERRORS>
          <IFOUTOCTETS>520452463</IFOUTOCTETS>
          <IFSPEED>10000000</IFSPEED>
          <IFSTATUS>1</IFSTATUS>
          <IFTYPE>24</IFTYPE>
          <IP>127.0.0.1</IP>
          <IPS>
            <IP>127.0.0.1</IP>
          </IPS>
        </PORT>
        <PORT>
          <CONNECTIONS>
            <CONNECTION>
              <MAC>xx:xx:xx:xx:xx:xx</MAC>
            </CONNECTION>
          </CONNECTIONS>
          <IFDESCR>WAN</IFDESCR>
          <IFINERRORS>527</IFINERRORS>
          <IFINOCTETS>1345673106</IFINOCTETS>
          <IFINTERNALSTATUS>1</IFINTERNALSTATUS>
          <IFLASTCHANGE>0.00 seconds</IFLASTCHANGE>
          <IFMTU>1500</IFMTU>
          <IFNAME>WAN</IFNAME>
          <IFNUMBER>2</IFNUMBER>
          <IFOUTERRORS>0</IFOUTERRORS>
          <IFOUTOCTETS>1235654894</IFOUTOCTETS>
          <IFSPEED>1000000000</IFSPEED>
          <IFSTATUS>1</IFSTATUS>
          <IFTYPE>6</IFTYPE>
          <IP>XXX.XXX.XXX.XXX</IP>
          <IPS>
            <IP>XXX.XXX.XXX.XXX</IP>
          </IPS>
          <MAC>xx:xx:xx:xx:xx:xx</MAC>
        </PORT>
        <PORT>
          <IFDESCR>eth1</IFDESCR>
          <IFINERRORS>162</IFINERRORS>
          <IFINOCTETS>731085552</IFINOCTETS>
          <IFINTERNALSTATUS>1</IFINTERNALSTATUS>
          <IFLASTCHANGE>0.00 seconds</IFLASTCHANGE>
          <IFMTU>1500</IFMTU>
          <IFNAME>eth1</IFNAME>
          <IFNUMBER>3</IFNUMBER>
          <IFOUTERRORS>0</IFOUTERRORS>
          <IFOUTOCTETS>3322126822</IFOUTOCTETS>
          <IFSPEED>1000000000</IFSPEED>
          <IFSTATUS>1</IFSTATUS>
          <IFTYPE>6</IFTYPE>
          <MAC>xx:xx:xx:xx:xx:xx</MAC>
        </PORT>
        <PORT>
          <IFDESCR>DMZ</IFDESCR>
          <IFINERRORS>0</IFINERRORS>
          <IFINOCTETS>0</IFINOCTETS>
          <IFINTERNALSTATUS>2</IFINTERNALSTATUS>
          <IFLASTCHANGE>0.00 seconds</IFLASTCHANGE>
          <IFMTU>1500</IFMTU>
          <IFNAME>DMZ</IFNAME>
          <IFNUMBER>4</IFNUMBER>
          <IFOUTERRORS>0</IFOUTERRORS>
          <IFOUTOCTETS>0</IFOUTOCTETS>
          <IFSPEED>0</IFSPEED>
          <IFSTATUS>2</IFSTATUS>
          <IFTYPE>6</IFTYPE>
          <MAC>xx:xx:xx:xx:xx:xx</MAC>
        </PORT>
        <PORT>
          <IFDESCR>DSL</IFDESCR>
          <IFINERRORS>0</IFINERRORS>
          <IFINOCTETS>0</IFINOCTETS>
          <IFINTERNALSTATUS>2</IFINTERNALSTATUS>
          <IFLASTCHANGE>0.00 seconds</IFLASTCHANGE>
          <IFMTU>1500</IFMTU>
          <IFNAME>DSL</IFNAME>
          <IFNUMBER>5</IFNUMBER>
          <IFOUTERRORS>0</IFOUTERRORS>
          <IFOUTOCTETS>0</IFOUTOCTETS>
          <IFSPEED>1000000000</IFSPEED>
          <IFSTATUS>2</IFSTATUS>
          <IFTYPE>6</IFTYPE>
          <MAC>xx:xx:xx:xx:xx:xx</MAC>
        </PORT>
        <PORT>
          <IFDESCR>LAN2</IFDESCR>
          <IFINERRORS>0</IFINERRORS>
          <IFINOCTETS>0</IFINOCTETS>
          <IFINTERNALSTATUS>2</IFINTERNALSTATUS>
          <IFLASTCHANGE>0.00 seconds</IFLASTCHANGE>
          <IFMTU>1500</IFMTU>
          <IFNAME>LAN2</IFNAME>
          <IFNUMBER>6</IFNUMBER>
          <IFOUTERRORS>0</IFOUTERRORS>
          <IFOUTOCTETS>0</IFOUTOCTETS>
          <IFSPEED>1000000000</IFSPEED>
          <IFSTATUS>2</IFSTATUS>
          <IFTYPE>6</IFTYPE>
          <MAC>xx:xx:xx:xx:xx:xx</MAC>
        </PORT>
        <PORT>
          <IFDESCR>LAN3</IFDESCR>
          <IFINERRORS>0</IFINERRORS>
          <IFINOCTETS>0</IFINOCTETS>
          <IFINTERNALSTATUS>2</IFINTERNALSTATUS>
          <IFLASTCHANGE>0.00 seconds</IFLASTCHANGE>
          <IFMTU>1500</IFMTU>
          <IFNAME>LAN3</IFNAME>
          <IFNUMBER>7</IFNUMBER>
          <IFOUTERRORS>0</IFOUTERRORS>
          <IFOUTOCTETS>0</IFOUTOCTETS>
          <IFSPEED>10000000</IFSPEED>
          <IFSTATUS>2</IFSTATUS>
          <IFTYPE>6</IFTYPE>
          <MAC>xx:xx:xx:xx:xx:xx</MAC>
        </PORT>
        <PORT>
          <IFDESCR>LAN4</IFDESCR>
          <IFINERRORS>0</IFINERRORS>
          <IFINOCTETS>0</IFINOCTETS>
          <IFINTERNALSTATUS>2</IFINTERNALSTATUS>
          <IFLASTCHANGE>0.00 seconds</IFLASTCHANGE>
          <IFMTU>1500</IFMTU>
          <IFNAME>LAN4</IFNAME>
          <IFNUMBER>8</IFNUMBER>
          <IFOUTERRORS>0</IFOUTERRORS>
          <IFOUTOCTETS>0</IFOUTOCTETS>
          <IFSPEED>10000000</IFSPEED>
          <IFSTATUS>2</IFSTATUS>
          <IFTYPE>6</IFTYPE>
          <MAC>xx:xx:xx:xx:xx:xx</MAC>
        </PORT>
        <PORT>
          <IFDESCR>LAN5</IFDESCR>
          <IFINERRORS>0</IFINERRORS>
          <IFINOCTETS>0</IFINOCTETS>
          <IFINTERNALSTATUS>2</IFINTERNALSTATUS>
          <IFLASTCHANGE>0.00 seconds</IFLASTCHANGE>
          <IFMTU>1500</IFMTU>
          <IFNAME>LAN5</IFNAME>
          <IFNUMBER>9</IFNUMBER>
          <IFOUTERRORS>0</IFOUTERRORS>
          <IFOUTOCTETS>0</IFOUTOCTETS>
          <IFSPEED>10000000</IFSPEED>
          <IFSTATUS>2</IFSTATUS>
          <IFTYPE>6</IFTYPE>
          <MAC>xx:xx:xx:xx:xx:xx</MAC>
        </PORT>
        <PORT>
          <IFDESCR>LAN6</IFDESCR>
          <IFINERRORS>0</IFINERRORS>
          <IFINOCTETS>0</IFINOCTETS>
          <IFINTERNALSTATUS>2</IFINTERNALSTATUS>
          <IFLASTCHANGE>0.00 seconds</IFLASTCHANGE>
          <IFMTU>1500</IFMTU>
          <IFNAME>LAN6</IFNAME>
          <IFNUMBER>10</IFNUMBER>
          <IFOUTERRORS>0</IFOUTERRORS>
          <IFOUTOCTETS>0</IFOUTOCTETS>
          <IFSPEED>10000000</IFSPEED>
          <IFSTATUS>2</IFSTATUS>
          <IFTYPE>6</IFTYPE>
          <MAC>xx:xx:xx:xx:xx:xx</MAC>
        </PORT>
        <PORT>
          <CONNECTIONS>
            <CONNECTION>
			  <MAC>xx:xx:xx:xx:xx:xx</MAC>
			  <MAC>xx:xx:xx:xx:xx:xx</MAC>
			  <MAC>xx:xx:xx:xx:xx:xx</MAC>
			  <MAC>xx:xx:xx:xx:xx:xx</MAC>
			  <MAC>xx:xx:xx:xx:xx:xx</MAC>
            </CONNECTION>
          </CONNECTIONS>
          <IFDESCR>LAN1</IFDESCR>
          <IFINERRORS>0</IFINERRORS>
          <IFINOCTETS>629069860</IFINOCTETS>
          <IFINTERNALSTATUS>1</IFINTERNALSTATUS>
          <IFLASTCHANGE>0.00 seconds</IFLASTCHANGE>
          <IFMTU>1500</IFMTU>
          <IFNAME>LAN1</IFNAME>
          <IFNUMBER>11</IFNUMBER>
          <IFOUTERRORS>0</IFOUTERRORS>
          <IFOUTOCTETS>3563711805</IFOUTOCTETS>
          <IFSPEED>10000000</IFSPEED>
          <IFSTATUS>1</IFSTATUS>
          <IFTYPE>6</IFTYPE>
          <IP>XXX.XXX.XXX.XXX</IP>
          <IPS>
            <IP>XXX.XXX.XXX.XXX</IP>
          </IPS>
          <MAC>xx:xx:xx:xx:xx:xx</MAC>
        </PORT>
        <PORT>
          <IFDESCR>vpnt1</IFDESCR>
          <IFINERRORS>0</IFINERRORS>
          <IFINOCTETS>0</IFINOCTETS>
          <IFINTERNALSTATUS>1</IFINTERNALSTATUS>
          <IFLASTCHANGE>0.00 seconds</IFLASTCHANGE>
          <IFMTU>1500</IFMTU>
          <IFNAME>vpnt1</IFNAME>
          <IFNUMBER>12</IFNUMBER>
          <IFOUTERRORS>0</IFOUTERRORS>
          <IFOUTOCTETS>0</IFOUTOCTETS>
          <IFSPEED>0</IFSPEED>
          <IFSTATUS>1</IFSTATUS>
          <IFTYPE>131</IFTYPE>
        </PORT>
      </PORTS>
    </DEVICE>
    <MODULEVERSION>3.1</MODULEVERSION>
    <PROCESSNUMBER>1</PROCESSNUMBER>
  </CONTENT>
  <DEVICEID>foo</DEVICEID>
  <QUERY>SNMPQUERY</QUERY>
</REQUEST>
```
[debug] cleaning 1 worker threads

**Comment by Geo-Ron** [comment]
Cleaned anmp output

**Comment by g-bougard** [comment]
Hi @Geo-Ron,

`full match for sysobjectID .1.3.6.1.4.1.8072.3.2.10 in database` means `.1.3.6.1.4.1.8072.3.2.10` is still known.
Looking in the file, `8072.3.2.10` matches `8072.3.2.10 Net-SNMP COMPUTER Linux`.
A lot of different devices are telling they are linux computer and then only specific can fix such kind of problem.
So this is the wrong place to open such issue. You should open a new one on [fusioninventory-agent issues](https://github.com/fusioninventory/fusioninventory-agent/issues), telling which agent version you're using (it seems you're using latest 2.4.2, but you need to confirm that). You need also to explain what is wrong in the identification as "_The device is not correctly identified._" is really not clear on what is wrong.
You should know Checkpoint was added in 2.4.2 to fix [fusioninventory-agent #480 issue](https://github.com/fusioninventory/fusioninventory-agent/issues/480) but we have missed something.

Finally for such kind of issue we really need a full [snmpwalk output](http://fusioninventory.org/documentation/bugreport/#snmpwalk-output). Before submitting, you can of course obfuscate any private data not related to the issue, like ips, macs or contact informations.

I'm closing this one as not related to sysobject.ids, but don't forget to link it to the new one.

Thank you for your involving.
